src/lib/ast/empty.ts

```diff
@@ -0,0 +1,47 @@
+import { Skill } from "../types";
+import { SkillExpression } from "./skillExpression";
+import { Variable } from "./variable";
+import { SkillsRelations } from "./skillsRelation";
+
+/*
+ * An empty skill expression used in case the learning skill does not have any requirement
+ * The evaluation of this skill expression is always true
+ * We use this to ensure consistency across all learning units
+ * @param terms An empty set of the variables
+ * @author Alamoush
+ */
+export class Empty extends SkillExpression {
+	type: string;
+
+	constructor(private terms: SkillExpression[]) {

```

Comment:
Is the parameter terms realy needed for an Empty condition?

Reply:
I removed the parameter terms.